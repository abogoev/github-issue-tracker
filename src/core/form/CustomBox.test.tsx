import React from "react";
import CustomBox from "./CustomBox";
import { create } from "react-test-renderer";
import { Text } from "react-native";

describe("CustomBox", () => {
  it("should render correctly", () => {
    const randomText = "Hello";
    const component = create(
      <CustomBox>
        <Text>{randomText}</Text>
      </CustomBox>
    );

    component.root.findByType(Text).props.children.toBe(randomText);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
