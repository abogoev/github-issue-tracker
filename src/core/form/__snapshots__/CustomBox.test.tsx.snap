// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomBox should render correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "rgb(246, 248, 250)",
        "borderColor": "rgb(216, 222, 228)",
        "borderRadius": 6,
        "borderWidth": 1,
        "padding": 16,
      },
      undefined,
    ]
  }
>
  <Text>
    Hello
  </Text>
</View>
`;
